DOC-777: schemes and ports are ignored (#1975)
tinymce/tinymce-docs: enterprise/server/configure.md

The `allowed-origins` element configures a list of **all** values that can be expected by the server-side components in a HTTP Origin header from your {{site.productname}} instances (see the [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) for more information on the HTTP Origin header). In short, you'll need to supply a list of all the URLs that your {{site.productname}} instances will be served from without the path information.

> > **Note:** Unless defining regular expression based `allowed-origins`, the `scheme` (e.g. http) and the `port` (e.g. 800) are **ignored**. 

Reviewer: ```suggestion
> **Important:** Unless defining regular expression based `allowed-origins`, the `scheme` (e.g. http) and the `port` (e.g. 800) are **ignored**. 
```

Author: I changed Note -> Important

This is best illustrated with some examples:

If users load {{site.productname}} from the following URLs:

- `http://server.example.com/editor.php`
- `http://server.example.com/subpage/editor.php`

Add `server.example.com` to the `allowed-origins` list.

If users load {{site.productname}} from the following URLs:

- `https://server.example.com/editor.php`
- `http://server.example.com/subpage/editor.php`

> Add `server.example.com` to the `allowed-origins` list because the scheme is ignored.

Reviewer 2: This wording is a little clumsy.

Author: I agree. Thoughts?

Reviewer: ```suggestion
Add `server.example.com` to the `allowed-origins` list (The scheme is ignored by the service).
```
?

Author: I thought about putting it in brackets as well

Author: This is pretty urgent. How much do you care?

Reviewer 2: You know me well enough to know my answer to that question

{{site.requires_jsscwar_230v}}

> Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` is not excluded from the matching process.

Reviewer: ```suggestion
Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` are not excluded from the matching process.
```

Author: Entire sentence was deleted, because it was wrong.

> Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` is not excluded from the matching process.

Author: Not correct after testing. Revising...

Author: Nope. Completely wrong. Fixing.

Author: Regular expressions have the same limitation. Only match the host section. I've updated the docs to match this.
